homeassistant/util/package.py

```diff
@@ -28,8 +28,13 @@ def is_virtual_env() -> bool:
 
 @cache
 def is_docker_env() -> bool:
-    """Return True if we run in a docker env."""
-    return Path("/.dockerenv").exists()
+    """Return True if we run in a container env."""
+    return (
+        Path("/run/.containerenv").exists()
+        or Path("/.dockerenv").exists()
+        or os.environ.get("KUBERNETES_SERVICE_HOST")
+        or is_official_image()
```

Comment:
@candrews an import seems to be missing?

../Frenck

Reply:
Added it: https://github.com/candrews/core-1/commit/ba7985928fe3f5f12b3aecf5b7627820d4d0d516